week7/src/LinkedHashSetExample.java

```diff
@@ -0,0 +1,51 @@
+import java.util.LinkedHashSet;
+import java.util.Scanner;
+
+public class LinkedHashSetExample {
+    public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
+        int T = input.nextInt();
+        while (T-- > 0) {
+            int Q = input.nextInt();
+            LinkedHashSet<Integer> linkedHashSet = new LinkedHashSet<>();
+            while (Q-- > 0) {
+                String query = input.next();
+                switch (query) {
+                    case "a":
+                        int x = input.nextInt();
+                        linkedHashSet.add(x);
+                        break;
+                    // I merged case b and case f because They have the same implementation
```

Comment:
b and f are not the same for b you print the values in hashset in increasing order but for f you print them in the order they are inserted first. 
for example if you add 5 2 1 
f will print 5 2 1
b will print 1 2 5